data/json/recipes/recipe_food.json

```diff
@@ -4256,6 +4256,30 @@
   ]
 },{
   "type" : "recipe",
+  "result": "syrup",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_DRINKS",
+  "skill_used": "cooking",
+  "difficulty": 3,
+  "time": 40000,
+  "autolearn": true,
+  "batch_time_factors": [ 60, 3 ],
+  "qualities":[
+    {"id":"BOIL","level":2}
+  ], "tools": [
+    [
+      [ "hotplate", 8 ],
+      [ "toolset", 8 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "maple_sap", 35 ]
```

Comment:
A single maple tap will produce around 224 charges in the producing season assuming there is a big enough container or that sap is harvested regularly. That means that each tapped maple tree have the possibility to produce a little more than 6 charges of maple syrup which should be balanced enough for now.